src/app/public/modules/colorpicker/colorpicker.component.ts

```diff
@@ -84,16 +89,65 @@ export class SkyColorpickerComponent implements OnInit, OnDestroy {
   public initialColor: string;
   public lastAppliedColor: string;
   public isVisible: boolean;
```

Comment:
I know you didn't touch this, but should we rename this to something more specific like `isPickerVisible`? I had to refer back to this in the template several times just to remember what it was for. I know its a public property, but consumers should not be accessing this, right?

Reply:
Good idea.